docker-compose-firecrawl.yml

```diff
@@ -0,0 +1,96 @@
+name: firecrawl
+services:
+# Firecrawl services
+  playwright-service:
+    image: trieve/playwright-service-ts:latest
+    environment:
+      - PORT=3000
+      - PROXY_SERVER=${PROXY_SERVER}
+      - PROXY_USERNAME=${PROXY_USERNAME}
+      - PROXY_PASSWORD=${PROXY_PASSWORD}
+      - BLOCK_MEDIA=${BLOCK_MEDIA}
+    networks:
+      - backend
+
+  firecrawl-api:
+    image: trieve/firecrawl:latest
```

Comment:
Are we making a custom firecrawl image?

Reply:
yes already made it

Reply:
Ah gotcha, its just firecrawl hasn't published one.